packages/plugins/plugin-transcription/src/capabilities/app-graph-builder.ts

```diff
@@ -98,11 +110,33 @@ export default (context: PluginsContext) =>
               schema: TranscriptType,
               getIntent: ({ space }: { space: Space }) =>
                 createIntent(TranscriptionAction.Create, { spaceId: space.id }),
-              onJoin: () => {
-                const transcriptionManager = new TranscriptionManager(client.edge);
+
+              // TODO(burdon): Is this the right place to inisitalize the state? Change to a capability?
```

Comment:
this could (maybe should) be an intent - there's already a capability which is storing the state, but as the logic gets more complex it would be better to have it factored out of the graph builder and into the intent resolver